cvpaperchallenge-alumni/limit-lab-homepage
# Enhancement/provide recent news image

## Issue URL

close: #10

## Change overview

- Add the image for "Recent News" section
- Add cvpaper.challenge logo to the footer
- Tweak the top page layout
- Reconsider the folder structure (move design part under `src/components` and move web page data contents under `src/data`)

## How to test

Check out the top page design, layout, animation in your eyes and make sure it works properly

## Note for reviewers

> [!Important]
> As I mentioned in #2, the responsive design has not been implemented for now.

> [!Note]
> I came up with a different idea for presenting "Recent News" images. The idea is to provide the images as preserved data in a [carousel component](https://ui.shadcn.com/docs/components/carousel), displaying them iteratively every n seconds, with the corresponding description highlighted on the right side.
> However, the current design isn't bad (and it feels like a waste to discard it after spending so much time on it, haha), so I'd like to keep it as is for now and focus on other priority tasks.
